amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/exceptions/CustomerApplicationException.java

```diff
@@ -0,0 +1,10 @@
+package software.amazon.kinesis.leases.exceptions;
+
+public class CustomerApplicationException extends Exception {
```

Comment:
For every new src file add copyright blob. Also add @KinesisClientInternalApi annotation